src/ProjectOrigin.WalletSystem.IntegrationTests/ReceiveSliceTests.cs

```diff
@@ -42,16 +57,23 @@ public async Task ReceiveSlice()
                 RandomR = ByteString.CopyFrom(new byte[] { 0x01, 0x02, 0x03, 0x04 }),
             };
 
+            var harness = _grpcFixture.GetRequiredService<ITestHarness>();
+
             //Act
-            var response = await client.ReceiveSliceAsync(request);
+            await client.ReceiveSliceAsync(request);
 
             //Assert
-            using (var connection = new NpgsqlConnection(_dbFixture.ConnectionString))
-            {
-                // Verify slice created in database
-                var slice = await connection.QueryFirstOrDefaultAsync<ReceivedSlice>("SELECT * FROM ReceivedSlices WHERE certificateId = @id", new { id = certId });
```

Comment:
Is there a migration script for removing the ReceivedSlices table? Do we need that table anymore?

Reply:
Actually no, I have also had the thought, but probably we also need a migration to empty the table into MessageBroker.

Reply:
implemented